README.md

```diff
@@ -9,7 +9,7 @@ This repo houses the assets used to build the website for Longhorn, available at
 To run the website locally, you need to have the [Hugo](https://gohugo.io) static site generator installed (installation instructions [here](https://gohugo.io/getting-started/installing/)). Once Hugo is installed:
 
 ```bash
-npm install bulma
+yarn
```

Comment:
Just `yarn` is good enough? I don't know much about `npm` vs `yarn` though.

Reply:
Because the package.json and yarn.lock exist, _yarn_ will look up the package.json and install the dependency. 

Command "yarn" or "yarn install" both work, ref to https://yarnpkg.com/lang/en/docs/usage/